Xext/xf86bigfont.c

```diff
@@ -511,7 +512,11 @@ ProcXF86BigfontQueryFont(ClientPtr client)
             }
             if (pDesc && !badSysCall) {
                 *(CARD32 *) (pCI + nCharInfos) = signature;
+#ifdef XFONT2
+                if (!xfont2_font_set_private(pFont, FontShmdescIndex, pDesc)) {
```

Comment:
this could go into the header too

Reply:
I didn't put this in the header because FontSetPrivate, the 'symbol' I'm ifdef-ing is actually a macro that can be called with libXfont2 too, but it's the wrong macro to call when using libXfont2.

I can #undef it in the header and redefine it to xfont2_font_set_private.
Should I do that?

Reply:
that would look cleaner, i guess

Reply:
done